concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-177	177	hard	HARD	1884	1	3.9047619047619	4.71428571428571	1.28571428571429	0.095238095238095	1	1.4108423691101	0.462910049886275	1.55379719213471	0.300792603759119	1.22474487139159	Haptic	0.41991341991342	228	5.43372200355424	4
+Lynott-2009-423-178	178	harsh			1	3.19047619047619	2.52380952380952	3.33333333333333	1.80952380952381	2.28571428571429	1.5039630187956	1.40068010691405	1.59164485150844	1.86061945649957	1.67758329578168	Auditory	0.115942028985507	15	2.77258872223978	5
+Lynott-2009-423-179	179	hazy			1	4.38095238095238	0.142857142857143	0.952380952380952	0.476190476190476	0	0.973457265430305	0.478091443733757	1.07126982951031	0.872871560943969	0	Visual	0.736	2	1.09861228866811	4
+Lynott-2009-423-180	180	heavy	HEAVY	1210	1	3.52380952380952	3.9047619047619	1.71428571428571	0.238095238095238	0.571428571428571	1.40068010691405	1.67047184665776	1.79284291400159	0.768424485864545	1.32557265253281	Haptic	0.368421052631579	94	4.55387689160054	5
+Lynott-2009-423-181	181	herby			1	2.80952380952381	0.761904761904762	0.047619047619048	4.14285714285714	4.80952380952381	1.32736760616822	1.17918454470714	0.218217890235992	0.792824967172091	0.402373908081478	Gustatory	0.378787878787879	0	0	5
+Lynott-2009-423-182	182	high	HIGH	1265	1	4.38095238095238	2.28571428571429	2.47619047619048	0.285714285714286	0.285714285714286	0.920662287496913	2.07708587070581	1.86061945649957	0.783763812819726	0.783763812819726	Visual	0.42156862745098	383	5.95064255258773	4
+Lynott-2009-423-183	183	hissing			1	2.0952380952381	0.333333333333333	4.52380952380952	0.047619047619048	0.047619047619048	1.48002573980191	0.795822425754221	0.928388260322567	0.218217890235992	0.218217890235992	Auditory	0.635135135135135	2	1.09861228866811	7
+Lynott-2009-423-184	184	hoarse			0.952380952380952	1.55	1.45	3.9	0.15	0.35	1.73129096949433	1.57195821559574	1.74416319854476	0.489360484929593	0.988086934168084	Auditory	0.506756756756757	2	1.09861228866811	6
+Lynott-2009-423-185	185	hollow			1	4.38095238095238	4.14285714285714	3.85714285714286	0.095238095238095	0.571428571428571	0.804747816162957	1.01418510567422	1.55838744494796	0.300792603759119	1.28730060869358	Visual	0.328467153284672	11	2.484906649788	6
+Lynott-2009-423-186	186	honeyed			1	3.04761904761905	1.33333333333333	0.333333333333333	3.57142857142857	4.09523809523809	1.62715059156153	1.39044357430761	0.795822425754221	1.32557265253281	1.37494588638106	Gustatory	0.303846153846154	0	0	7
+Lynott-2009-423-187	187	hot	HOT	1286	1	3.33333333333333	4.85714285714286	1.04761904761905	1.66666666666667	4.42857142857143	1.42594997574716	0.358568582800317	1.39557122627942	2.08166599946613	1.39897921969658	Haptic	0.248447204968944	91	4.52178857704904	3
+Lynott-2009-423-188	188	howling	HOWLING	2523	1	2.80952380952381	0.142857142857143	4.95238095238095	0.142857142857143	0.142857142857143	1.4359334113756	0.654653670707977	0.218217890235994	0.654653670707977	0.654653670707977	Auditory	0.587209302325581	3	1.38629436111989	7
```

Comment:
The definition of HOWLING could benefit from disambiguation, e.g. contrasted with howling wind.

Reply:
Also, our HOWLING seems to be a noun.